src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -261,6 +261,46 @@ public override FormulaValue Visit(UnaryOpNode node, SymbolContext context)
             return CommonErrors.UnreachableCodeError(node.IRContext);
         }
 
+        public override FormulaValue Visit(AggregateCoercionNode node, SymbolContext context)
+        {
+            var arg1 = node.Child.Accept(this, context);
+
+            if (node.Op == UnaryOpKind.TableToTable)
+            {
+                var table = (TableValue)arg1;
+                var tableType = (TableType)node.IRContext.ResultType;
+                var resultRows = new List<DValue<RecordValue>>();
+                foreach (var row in table.Rows)
+                {
+                    if (row.IsValue)
+                    {
+                        List<NamedValue> fields = new List<NamedValue>();
+                        foreach (var coercion in node.FieldCoercions)
+                        {
+                            var record = row.Value;
+                            var newScope = context.WithScope(node.Scope).WithScopeValues(record);
```

Comment:
you can pull `var scopeContext = context.WithScope(node.Scope)` out of the loop, and reuse it.

`var newScope = scopeContext.WithScopeValues(record)`

Reply:
Done